<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>358</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Replace"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionSep_into"/>
    <addaction name="actionStep_Out"/>
    <addaction name="actionStep_Over"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_Breakpoint"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Bookmarks</string>
    </property>
    <widget class="QMenu" name="menuBookmarks_2">
     <property name="title">
      <string>Bookmarks</string>
     </property>
     <addaction name="actionBookmark1"/>
    </widget>
    <addaction name="actionToggle_Bookmark"/>
    <addaction name="actionDelete_Bookmark"/>
    <addaction name="menuBookmarks_2"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionConfigure_Toolbars"/>
    <addaction name="actionConfigure_Shortcuts"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="actionIndex"/>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="actionRecent_file1"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionRebuild"/>
    <addaction name="actionClean"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuDebug"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_All"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QToolBar" name="buildToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBuild"/>
   <addaction name="actionRebuild"/>
   <addaction name="actionClean"/>
   <addaction name="actionRun"/>
  </widget>
  <widget class="QToolBar" name="debugToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionSep_into"/>
   <addaction name="actionStep_Out"/>
   <addaction name="actionStep_Over"/>
   <addaction name="actionStop"/>
   <addaction name="actionToggle_Breakpoint"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/filenew.png</normaloff>:/icons/icons/filenew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/fileopen.png</normaloff>:/icons/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>_</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/filesave.png</normaloff>:/icons/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/filesaveall.png</normaloff>:/icons/icons/filesaveall.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="toolTip">
    <string>Save All Open Documents</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/filesaveas.png</normaloff>:/icons/icons/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As ...</string>
   </property>
   <property name="toolTip">
    <string>Save current document with specifient name</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/filePrintPreview.png</normaloff>:/icons/icons/filePrintPreview.png</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/fileprint.png</normaloff>:/icons/icons/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Print ...</string>
   </property>
   <property name="toolTip">
    <string>Prints the current document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/editundo.png</normaloff>:/icons/icons/editundo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>undo last action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/editredo.png</normaloff>:/icons/icons/editredo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo last undo action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/editcopy.png</normaloff>:/icons/icons/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy selected text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/editcut.png</normaloff>:/icons/icons/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut selected text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/editpaste.png</normaloff>:/icons/icons/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste text from clipboard to editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/editdelete.png</normaloff>:/icons/icons/editdelete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Selected Text</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/edit-select-all.png</normaloff>:/icons/icons/edit-select-all.png</iconset>
   </property>
   <property name="text">
    <string>Select All ...</string>
   </property>
   <property name="toolTip">
    <string>Select the current document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/edit-find-replace.png</normaloff>:/icons/icons/edit-find-replace.png</iconset>
   </property>
   <property name="text">
    <string>Find/Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/buildbuild.png</normaloff>:/icons/icons/buildbuild.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionRebuild">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/buildrebuild.png</normaloff>:/icons/icons/buildrebuild.png</iconset>
   </property>
   <property name="text">
    <string>Rebuild</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/buildclean.png</normaloff>:/icons/icons/buildclean.png</iconset>
   </property>
   <property name="text">
    <string>Clean ...</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/buildrun.png</normaloff>:/icons/icons/buildrun.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run the executable</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/debugger_run.png</normaloff>:/icons/icons/debugger_run.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start debugging</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSep_into">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/debug-step-into.png</normaloff>:/icons/icons/debug-step-into.png</iconset>
   </property>
   <property name="text">
    <string>Step into</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionStep_Out">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/debug-step-out.png</normaloff>:/icons/icons/debug-step-out.png</iconset>
   </property>
   <property name="text">
    <string>Step Out</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionStep_Over">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/debug-step-over.png</normaloff>:/icons/icons/debug-step-over.png</iconset>
   </property>
   <property name="text">
    <string>Step Over</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionToggle_Breakpoint">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/editbreak.png</normaloff>:/icons/icons/editbreak.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Breakpoint</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/media-playback-stop.png</normaloff>:/icons/icons/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/helphome.png</normaloff>:/icons/icons/helphome.png</iconset>
   </property>
   <property name="text">
    <string>Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/helpkeyword.png</normaloff>:/icons/icons/helpkeyword.png</iconset>
   </property>
   <property name="text">
    <string>Index</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/helpgreettings.png</normaloff>:/icons/icons/helpgreettings.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/form.png</normaloff>:/icons/icons/form.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/helpqt.png</normaloff>:/icons/icons/helpqt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionToggle_Bookmark">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/editadd.png</normaloff>:/icons/icons/editadd.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionDelete_Bookmark">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/editclear.png</normaloff>:/icons/icons/editclear.png</iconset>
   </property>
   <property name="text">
    <string>Delete Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/configure.png</normaloff>:/icons/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionConfigure_Toolbars">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/configure-toolbars.png</normaloff>:/icons/icons/configure-toolbars.png</iconset>
   </property>
   <property name="text">
    <string>Configure Toolbars...</string>
   </property>
  </action>
  <action name="actionConfigure_Shortcuts">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/configure-shortcuts.png</normaloff>:/icons/icons/configure-shortcuts.png</iconset>
   </property>
   <property name="text">
    <string>Configure Shortcuts...</string>
   </property>
  </action>
  <action name="actionRecent_file1">
   <property name="text">
    <string>recent file1</string>
   </property>
  </action>
  <action name="actionBookmark1">
   <property name="text">
    <string>Bookmark1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
